templates/aes-agent.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if and .Values.agent.enabled .Values.enableAES }}
+{{- if not (semver .Values.image.tag | (semver "1.12.0").Compare) }}
```

Comment:
We'll be merging https://github.com/datawire/ambassador-chart/pull/183 first, so I guess these changes will become a no-op, right?

Reply:
correct.